wordpress-mobile/gutenberg-mobile
# Fixes a refresh issue in iOS when merging blocks.

### Description:

Fixes a refresh issue in iOS when merging blocks.

Moves #167 forward.

### Related PRs:

Gutenberg PR: https://github.com/WordPress/gutenberg/pull/11220

### Testing:

Repeat the following process several times and make sure the resulting block shows the expected content:

1. Make sure you have 2 blocks of the same type (paragraph x 2 or heading x2).
2. Place the caret at the beginning of the bottom block.
3. Press backspace.
4. Make sure the resulting block is as expected and refreshed properly.
